README.md

```diff
@@ -77,6 +77,20 @@ The first time you set up the application, database tables need to be created:
 
 Web server should now be accessible at **http://localhost:80/**.
 
+## Translations
+
+### Extracting strings
+
+Once the docker is up, run:
+
+`$ docker-compose -f docker/docker-compose.dev.yml run web python manage.py extract_strings`
+
+### Compiling the strings
+
+The POT files are compiled automatically every time the docker is created, but in case you make any changes
```

Comment:
Again, what's "the docker"? Are you trying to describe a specific CritiqueBrainz container?